targets/TARGET_STM/TARGET_STM32WB/TARGET_STM32WB5MxG/TARGET_DISCO_WB5MMG/Rgb_led.cpp

```diff
@@ -0,0 +1,141 @@
+/* mbed Microcontroller Library
+ * SPDX-License-Identifier: BSD-3-Clause
+ ******************************************************************************
+ *
+ * Copyright (c) 2021 STMicroelectronics.
+ * All rights reserved.
+ *
+ * This software component is licensed by ST under BSD 3-Clause license,
+ * the "License"; You may not use this file except in compliance with the
+ * License. You may obtain a copy of the License at:
+ *                        opensource.org/licenses/BSD-3-Clause
+ *
+ ******************************************************************************
+ */
+
+/* application example
+
+#include "mbed.h"
+
+int main()
+{
+    while (true) {
+        rgb_led_off();
+        ThisThread::sleep_for(2s);
+
+        rgb_led_red();
+        ThisThread::sleep_for(2s);
+
+        rgb_led_green();
+        ThisThread::sleep_for(2s);
+
+        rgb_led_blue();
+        ThisThread::sleep_for(2s);
+
+        rgb_led_on(0x41, 0x41, 0);    // yellow
+        ThisThread::sleep_for(2s);
+
+        rgb_led_on(0x41, 0, 0x41);    // magenta
+        ThisThread::sleep_for(2s);
+
+        rgb_led_on(0, 0x41, 0x41);    // cyan
+        ThisThread::sleep_for(2s);
+
+        rgb_led_on(0x41, 0x41, 0x41); // white
+        ThisThread::sleep_for(2s);
+    }
+}
+*/
+
+#include "drivers/DigitalOut.h"
+#include "platform/mbed_wait_api.h"
+
+
+#define DELAY          1
+#define T_CYCLE_0      4
+#define T_CYCLE_1      1
+#define WRITE_COMMAND  0x3A
+
+#define rgb_led_Wait(NbCycles) wait_us(NbCycles * 10)
+
+
+void rgb_led_SendBit(uint8_t bit)
+{
+    mbed::DigitalOut RGB_LED_SDI(RGB_LED);
+
+    /* Start next cycle */
+    RGB_LED_SDI = 1;
+    rgb_led_Wait(DELAY);
+    RGB_LED_SDI = 0;
+    rgb_led_Wait(DELAY);
+
+    if (bit) {
+        RGB_LED_SDI = 1;
+        rgb_led_Wait(DELAY);
+        RGB_LED_SDI = 0;
+        rgb_led_Wait(T_CYCLE_1);
+    } else {
+        rgb_led_Wait(T_CYCLE_0);
+    }
+}
+
+
+void rgb_led_SendByte(uint8_t byte)
```

Comment:
some functions still use camelcase, should these be fixed as well? They are not private (I dont see static declarations here)?

Reply:
camelcase removed